src/PowerShellEditorServices.Protocol/LanguageServer/GetCommandsRequest.cs

```diff
@@ -0,0 +1,54 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Microsoft.PowerShell.EditorServices.Protocol.MessageProtocol;
+using System.Collections.Generic;
+using System.Management.Automation;
+
+namespace Microsoft.PowerShell.EditorServices.Protocol.LanguageServer
+{
+    /// <summary>
+    /// Defines a class that describes the request to get all PowerShell Commands from the current session
```

Comment:
Style nit: "Defines a class that describes" is implicit; the comment should say what the code doesn't. Ditto for other comments below